Sources/protoc-gen-swift/MessageGenerator.swift

```diff
@@ -53,7 +53,7 @@ class MessageGenerator {
     // when the message has one or more oneof{}s. As that will efficively
     // reduce the real number of fields and the message might not need heap
     // storage yet.
-    let useHeapStorage = isAnyMessage || descriptor.fields.count > 16 || hasSingleMessageField(descriptor: descriptor)
```

Comment:
looks like hasSingleMessageField didn't get removed (since it isn't called any more)

Reply:
Fixed.